Vagrantfile

```diff
@@ -39,6 +36,7 @@ Vagrant.configure("2") do |config|
     apt-get update
     apt-get install -y wget unattended-upgrades
     apt-get install -y docker.io docker-compose
+    usermod -a -G docker vagrant
```

Comment:
It doesn't look like you've based your commit on the latest version of this file. In the originale file "vagrant provision" and "vagrant up" was split, and this wasn't needed. 

P.S.: Docker command should always be run as root, since it basically gives all containers potentially root access to a system.